R/lincomb.R

```diff
@@ -1,8 +1,8 @@
 #' Linear Combination Filter
 #'
-#' `step_lincomb` creates a *specification* of a recipe
-#'  step that will potentially remove numeric variables that have
-#'  linear combinations between them.
+#' `step_lincomb()` creates a *specification* of a recipe step that will
+#' potentially remove numeric variables that have linear combinations between
+#' them.
```

Comment:
How about 

> "that are linear combinations of others."

?

I'm not familiar with the currently used verbiage for linear combinations.

Reply:
So, after reading up on this, it technically removed Linear Dependency. Linear combinations are when 2 or more vectors are combined in `v1 * a1 + v2 * a2 + ... + vn * an` with scalar `a`s.

Linear dependency is what happens when a linear combination is equal to 0.

In the following, we have that `c = a + b` so the variables have a linear combination between them. could also be written as `0 = a + b - c`.
```r
a <- c(0, 1, 1, 0)
b <- c(1, 0, 0, 1)
c <- c(1, 1, 1, 1)
```